inflection/tools/dictionary-parser/src/main/java/org/unicode/wikidata/ParseWikidata.java

```diff
@@ -72,33 +94,50 @@ void reset() {
             grammemes.clear();
             inflections.clear();
         }
-        private Lemma() {}
+
+        private Lemma() {
+        }
     }
 
     private final ParserOptions parserOptions;
     private final DocumentState documentState = new DocumentState();
     private final TreeSet<String> rareLemmas = new TreeSet<>();
     private final TreeSet<String> omitLemmas = new TreeSet<>();
+    private final Map<String, List<String>> mergeMap = new HashMap<>();
+    private final TreeSet<String> differ = new TreeSet<>();
+    private final Map<String, Pair<Lexeme, Integer>> lexemeMap = new HashMap<>();
 
-    ParseWikidata(ParserOptions parserOptions)
-    {
+    ParseWikidata(ParserOptions parserOptions) {
         this.parserOptions = parserOptions;
         for (var language : parserOptions.locales) {
             Properties rareLemmasProperties = new Properties();
-            String filePath = Paths.get(ParserDefaults.RESOURCES_DIR + "filter_" + language + ".properties").toAbsolutePath().toString();
+            String filePath = Paths.get(ParserDefaults.RESOURCES_DIR + "filter_" + language + ".properties")
+                    .toAbsolutePath().toString();
             try (var propertiesStream = new InputStreamReader(new FileInputStream(filePath), StandardCharsets.UTF_8)) {
                 rareLemmasProperties.load(propertiesStream);
                 for (var entry : rareLemmasProperties.entrySet()) {
                     String key = entry.getKey().toString();
                     String value = entry.getValue().toString();
-                    switch (value) {
-                        case "rare" : rareLemmas.add(key); break;
-                        case "omit" : omitLemmas.add(key); break;
-                        default: throw new IllegalArgumentException(key + ": Unknown key value " + value);
+                    if ("rare".equals(value)) {
+                        rareLemmas.add(key);
+                    } else if ("omit".equals(value)) {
+                        omitLemmas.add(key);
+                    } else if (value.startsWith("L")) {
```

Comment:
How about `value.matches("L[0-9]+")`? This ensures that the L is followed by numbers.